xbmc/windowing/tvos/WinSystemTVOS.h

```diff
@@ -51,6 +51,7 @@ class CWinSystemTVOS : public CWinSystemBase, public CRenderSystemGLES, public I
   bool DestroyWindow() override;
   bool ResizeWindow(int newWidth, int newHeight, int newLeft, int newTop) override;
   bool SetFullScreen(bool fullScreen, RESOLUTION_INFO& res, bool blankOtherDisplays) override;
+  int GetBufferAge() override { return 3; }
```

Comment:
is it possible to make this magic number something meaningful (create some enum class)?

Reply:
I've just moved the value. Previously, it was defined in the region tracker.

Rather than creating an enum, I would set it to NoOfBuffers(). But that's something for its own PR. Is that okay?

Reply:
Sure...